rs/rosetta-api/icp/tests/system_tests/test_cases/neuron_management.rs

```diff
@@ -1053,3 +1054,172 @@ fn test_stake_maturity() {
         );
     });
 }
+
+#[test]
+fn test_spawn_neuron() {
+    let mut runner = TestRunner::new(TestRunnerConfig {
+        max_shrink_iters: 0,
+        cases: *NUM_TEST_CASES,
+        ..Default::default()
+    });
+
+    runner
+        .run(
+            &(basic_identity_strategy().no_shrink()),
+            |controller_identity| {
+                let rt = Runtime::new().unwrap();
+                rt.block_on(async {
+                    let env = RosettaTestingEnvironment::builder()
+                        .with_initial_balances(
+                            vec![(
+                                AccountIdentifier::from(TEST_IDENTITY.sender().unwrap()),
+                                // A hundred million ICP should be enough
+                                icp_ledger::Tokens::from_tokens(100_000_000).unwrap(),
+                            )]
+                            .into_iter()
+                            .collect(),
+                        )
+                        .with_governance_canister()
+                        .build()
+                        .await;
+
+                    // Stake the minimum amount 100 million e8s
+                    let staked_amount = 100_000_000u64;
+                    let neuron_index = 0;
+
+                    env.rosetta_client
+                        .create_neuron(
+                            env.network_identifier.clone(),
+                            &(*TEST_IDENTITY).clone(),
+                            RosettaCreateNeuronArgs::builder(staked_amount.into())
+                                .with_neuron_index(neuron_index)
+                                .build(),
+                        )
+                        .await
+                        .unwrap();
+
+                    // See if the neuron was created successfully
+                    let agent = get_test_agent(env.pocket_ic.url().unwrap().port().unwrap()).await;
+                    let mut neuron = list_neurons(&agent).await.full_neurons[0].to_owned();
+                    assert_eq!(neuron.maturity_e8s_equivalent, 0);
+
+                    let new_maturity = 100_000_000_000;
+                    neuron.maturity_e8s_equivalent = new_maturity;
+                    update_neuron(&agent, neuron.into()).await;
+
+                    let neuron = list_neurons(&agent).await.full_neurons[0].to_owned();
+                    assert_eq!(neuron.maturity_e8s_equivalent, new_maturity);
+
+                    let percentage_to_stake = 50;
```

Comment:
Should this be `percentage_to_spawn`? Should we also test with an invalid percentage, similar as in `test_stake_maturity`?

Reply:
Done